bandit/reporters/info.h

```diff
@@ -274,6 +281,17 @@ namespace bandit {
         return std::string(2 * indentation_, ' ');
       }
 
+      void print_timing() {
+        if (report_timing_) {
+          std::chrono::duration<double> dur_in_sec(testcase_duration_);
+          stm_
+            << colorizer_.neutral()
```

Comment:
Not sure if this is the right color choice (well, that's about taste, I guess), so I will have to check later when I check out your PR branch to test it myself. :-)

Reply:
Here are some screenshots:
<img width="581" alt="Screenshot 2019-10-27 15 40 52" src="https://user-images.githubusercontent.com/879889/67636395-6e5d3900-f8d0-11e9-9895-1a8ea9e275a7.png">
<img width="720" alt="Screenshot 2019-10-27 15 42 09" src="https://user-images.githubusercontent.com/879889/67636397-73ba8380-f8d0-11e9-86ba-9a0229f73fe7.png">

Reply:
Your color settings are very non-standard in some way "[ PASS ]"and "Passed: <number>" is usually green.

Reply:
Yes indeed. That is why it misled me in this issue: https://github.com/banditcpp/bandit/issues/149